source/includes/_organizations.md

```diff
@@ -158,6 +176,51 @@ In order to interact with this endpoint you must pass an admin API key. Visit [o
   "org_label": "Algorithmia",
   "org_name": "algorithmiahq",
   "org_url": "https://algorithmia.com",
+  "external_admin_group": "sample external admin group",
+  "external_member_group": "sample external member group",
+  "external_id": "sampleExternalId",
+  "resource_type": "organization",
+  "self_link": "http://api.algorithmia.com/v1/organizations/algorithmiahq"
+}
+```
+
+A single [organization object](#the-organization-object) if a valid `org_name` was provided, otherwise an [error](#errors).
+
+## Get an organization through external ID
+
+```shell
+curl https://api.algorithmia.com/v1/organizations \
+  -H 'Authorization: Simple ADMIN_API_KEY'
+```
+
+```python
+# Getting an organization is not yet supported by our Python client library.
+```
+
+`GET /organizations/externalId/:external_id`
+
+### Authorization
+
+In order to interact with this endpoint you must pass an admin API key. Visit [our documentation](/developers/platform/customizing-api-keys#admin-api-keys) to learn more.
+
+### Query Parameters
+
+|Parameter|Type|Description|
+|-|-|-|
+|`external_id`|String|*Required*. The unique `external_id` which has been updated to match the ID given to this organization by the external identity provider.|
```

Comment:
let me know what you think here, if IDP is too caml specific we can of course call it something else

Reply:
I think instead of identifying via "external identity provider" we can say that it corresponds to the value set in our own update endpoint.  The ID can be any string, but users may find it useful for mapping to identifiers in external identity systems.